x-pack/plugins/uptime/public/components/monitor/monitor_charts.tsx

```diff
@@ -12,6 +12,7 @@ import { MonitorDuration } from './monitor_duration/monitor_duration_container';
 interface MonitorChartsProps {
   monitorId: string;
 }
+export const MONITOR_CHART_HEIGHT = '250px';
```

Comment:
We are working with an 8px grid at Elastic. Please make sure if you decide for a different height than 200 that it fits the grid.

Reply:
have changed it to 248px @katrin-freihofner